### PR TITLE
Update "Using the TCP Transport" Doc

### PR DESCRIPTION
**Pull request by chathuranga-jayanath-99**
## Purpose
Add separate endpoint in synapse configuration.

### CODE REVIEW

File: en/docs/learn/examples/transport-examples/tcp-transport-examples.md [hunk 1]
```diff
@@ -39,49 +39,62 @@ The following are the transport receiver properties.
...
-                </call> 
-                <log level="full"/> 
+=== "Proxy Service"
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <proxy name="TCPProxy" startOnLoad="true" transports="tcp" xmlns="http://ws.apache.org/ns/synapse">
+        <target>
+            <inSequence>
+                <property name="symbol" scope="default" type="STRING" value="IBM"/>
+                <enrich description="">
```
Comment (chanikag) [inline en/docs/learn/examples/transport-examples/tcp-transport-examples.md:49]: empty description.

Comment (chathuranga-jayanath-99) [reply en/docs/learn/examples/transport-examples/tcp-transport-examples.md:49]: by default when we create an enrich mediator it adds `description=""` even we didn't provide a description

File: en/docs/learn/examples/transport-examples/tcp-transport-examples.md [hunk 2]
```diff
@@ -277,14 +313,14 @@ Send the following message via TCP to the TCP listener port.
 <soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"><soapenv:Header/><soapenv:Body/></soapenv:Envelope>split<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"><soapenv:Header/><soapenv:Body/></soapenv:Envelope>
 ```
 
-In Linux, we can save the request in a <strong>request.xml</strong> file and use netcat to send the TCP request.
+In Mac, we can save the request in a <strong>request.xml</strong> file and use netcat to send the TCP request.
```
Comment (DinithiDiaz) [inline en/docs/learn/examples/transport-examples/tcp-transport-examples.md:316]:
```suggestion
On macOS, you can save the request in a `request.xml` file and use Netcat to send the TCP request.
```

File: en/docs/learn/examples/transport-examples/tcp-transport-examples.md [hunk 3]
```diff
@@ -192,9 +217,9 @@ Send the following message via TCP to the TCP listener port.
 ```xml
 <soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"><soapenv:Header/><soapenv:Body/></soapenv:Envelope>
 ```
-In Linux, we can save the request in a <strong>request.xml</strong> file and use netcat to send the TCP request. 
+In Mac, we can save the request in a <strong>request.xml</strong> file and use netcat to send the TCP request.
```
Comment (DinithiDiaz) [inline en/docs/learn/examples/transport-examples/tcp-transport-examples.md:220]:
```suggestion
On macOS, you can save the request in a `request.xml` file and use Netcat to send the TCP request. 
```

File: en/docs/learn/examples/transport-examples/tcp-transport-examples.md [hunk 4]
```diff
@@ -109,9 +122,9 @@ Send the following message via TCP to the TCP listener port.
 ```xml
 <soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"><soapenv:Header/><soapenv:Body/></soapenv:Envelope>|<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"><soapenv:Header/><soapenv:Body/></soapenv:Envelope>
 ```
-In Linux, we can save the request in a <strong>request.xml</strong> file and use netcat to send the TCP request. 
+In Mac, we can save the request in a <strong>request.xml</strong> file and use netcat to send the TCP request.
```
Comment (chanikag) [inline en/docs/learn/examples/transport-examples/tcp-transport-examples.md:125]:
```suggestion
In Linux, we can save the request in a <strong>request.xml</strong> file and use Netcat to send the TCP request. 
```

File: en/docs/learn/examples/transport-examples/tcp-transport-examples.md [hunk 5]
```diff
@@ -109,9 +122,9 @@ Send the following message via TCP to the TCP listener port.
 ```xml
 <soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"><soapenv:Header/><soapenv:Body/></soapenv:Envelope>|<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"><soapenv:Header/><soapenv:Body/></soapenv:Envelope>
 ```
-In Linux, we can save the request in a <strong>request.xml</strong> file and use netcat to send the TCP request. 
+In Mac, we can save the request in a <strong>request.xml</strong> file and use netcat to send the TCP request. 
 ```
-netcat localhost 6061 < request.xml
+nc localhost 6061 < request.xml
```
Comment (chanikag) [inline en/docs/learn/examples/transport-examples/tcp-transport-examples.md:127]:
```suggestion
netcat localhost 6061 < request.xml
```

File: en/docs/learn/examples/transport-examples/tcp-transport-examples.md [hunk 6]
```diff
@@ -192,9 +217,9 @@ Send the following message via TCP to the TCP listener port.
 ```xml
 <soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"><soapenv:Header/><soapenv:Body/></soapenv:Envelope>
 ```
-In Linux, we can save the request in a <strong>request.xml</strong> file and use netcat to send the TCP request. 
+On macOS, you can save the request in a `request.xml` file and use Netcat to send the TCP request.
```
Comment (chanikag) [inline en/docs/learn/examples/transport-examples/tcp-transport-examples.md:220]:
```suggestion
On Linux, you can save the request in a `request.xml` file and use Netcat to send the TCP request. 
```

File: en/docs/learn/examples/transport-examples/tcp-transport-examples.md [hunk 7]
```diff
@@ -192,9 +217,9 @@ Send the following message via TCP to the TCP listener port.
 ```xml
 <soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"><soapenv:Header/><soapenv:Body/></soapenv:Envelope>
 ```
-In Linux, we can save the request in a <strong>request.xml</strong> file and use netcat to send the TCP request. 
+On macOS, you can save the request in a `request.xml` file and use Netcat to send the TCP request. 
 ```
-netcat localhost 6061 < request.xml
+nc localhost 6061 < request.xml
```
Comment (chanikag) [inline en/docs/learn/examples/transport-examples/tcp-transport-examples.md:222]:
```suggestion
netcat localhost 6061 < request.xml
```

File: en/docs/learn/examples/transport-examples/tcp-transport-examples.md [hunk 8]
```diff
@@ -192,7 +217,7 @@ Send the following message via TCP to the TCP listener port.
 ```xml
 <soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"><soapenv:Header/><soapenv:Body/></soapenv:Envelope>
 ```
-In Linux, we can save the request in a <strong>request.xml</strong> file and use netcat to send the TCP request. 
+On Linux, you can save the request in a `request.xml` file and use Netcat to send the TCP request.
```
Comment (chanikag) [inline en/docs/learn/examples/transport-examples/tcp-transport-examples.md:220]:
```suggestion
In Linux, you can save the request in a `request.xml` file and use Netcat to send the TCP request. 
```

**Review by chanikag — APPROVED** [review]

**Pull request merged**